catalog/core/catalog-core-sourcemetricsplugin/src/main/java/ddf/catalog/metrics/source/SourceMetricsImpl.java

```diff
@@ -546,7 +543,6 @@ public SourceMetric(Metric metric, String sourceId, RrdJmxCollector collector) {
         public SourceMetric(Metric metric, String sourceId, RrdJmxCollector collector,
```

Comment:
If the `sourceId` field is no longer needed, we should remove the parameter from this ctor.